_research/ai-models/ai-urban-flows/2026-modeling-complex-urban-flows-gen-ai.md

```
@@ -0,0 +1,222 @@
+
+## References
+
+Francisco Giral, Álvaro Manzano, Ignacio Gómez, Petros Koumoutsakos, and Soledad Le Clainche.  
+**Generative Urban Flow Modeling: From Geometry to Airflow with Graph Diffusion.**  
+arXiv:2512.14725, 2025.  
+[https://arxiv.org/abs/2512.14725](https://arxiv.org/abs/2512.14725)
+
+Francisco Giral, Álvaro Manzano, Ignacio Gómez, Ricardo Vinuesa, and Soledad Le Clainche.  
+**GenDA: Generative Data Assimilation on Complex Urban Areas via Classifier-Free Diffusion Guidance.**  
+arXiv:2601.11440, 2026.  
+[https://arxiv.org/abs/2601.11440](https://arxiv.org/abs/2601.11440)
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)